src/background/services/paymentSession.ts

```diff
@@ -172,6 +179,13 @@ export class PaymentSession {
             }
           })
 
+          this.tabState.saveOverpaying(
+            this.tab,
+            this.url,
+            this.receiver,
+            this.intervalInMs
```

Comment:
Non overpaying sessions are getting saved as well. We should only save the overpaying ones.